java-client/src/main/java/com/cumulocity/sdk/client/ResponseParser.java

```diff
@@ -77,14 +83,32 @@ public void checkStatus(Response response, int... expectedStatusCodes) throws SD
 
     protected String getErrorMessage(Response response, int status) {
         String errorMessage = "Http status code: " + status;
+        String responseHeader = "";
 
         if (response.hasEntity()) {
             String errorRepresentation = getErrorRepresentation(response);
             if (errorRepresentation == null) {
                 errorRepresentation = NO_ERROR_REPRESENTATION;
             }
+            Map<String, List<String>> responseHeadermap = response.getStringHeaders();
```

Comment:
minor style comment - I would go for responseHeaderMap instead of responseHeadermap

Reply:
> minor style comment - I would go for responseHeaderMap instead of responseHeadermap

Thank you for the comment. In favor to pass the PR and not loosing approval, I will change it, when other comments arive in the PR process. ;)

Reply:
Sure. It is anyway just a suggestion :)